packages/docs-theme/src/components/documentation.tsx

```diff
@@ -87,6 +93,10 @@ export interface IDocumentationState {
 
 @HotkeysTarget
 export class Documentation extends React.PureComponent<IDocumentationProps, IDocumentationState> {
+    public static defaultProps: Partial<IDocumentationProps> = {
+        scrollParent: document.documentElement,
```

Comment:
@styu 🤔 i think i'd prefer to do this as default argument in the two functions below, rather than add `defaultProps`.